src/client/Client.cc

```diff
@@ -8438,6 +8455,29 @@ int Client::_open(Inode *in, int flags, mode_t mode, Fh **fhp,
     req->head.args.open.old_size = in->size;   // for O_TRUNC
     req->set_inode(in);
     result = make_request(req, perms);
+
+    /*
+     * Clients that have delegations enabled will wait on minimal caps, just
+     * to ensure that other clients holding delegations return theirs first.
```

Comment:
Please add an explanation to this comment on why we need to do this for posterity.